SparkPost/2web2ui
# FE-1161 - Fix content that was not properly wrapped in a <Panel/>

[FE-1161](https://sparkpost.atlassian.net/browse/FE-1161)

### What Changed
- Added wrapping `<Panel/>` where missing
- Updates `SendingDomainsTab` tests to remove reliance on snapshots for component testing
- Removes use of inline styling on the `<SendingDomainsTab/>` component in favor of `styled-components`

### How To Test
- Sign in to a user associated with the 108 account and navigate to `/account/subaccounts/101/sending-domains`
- Verify the content renders properly inside a `<Panel/>`
- Verify the empty state on the sending domains tab as well

### To Do
